docs/pyrona.md

```diff
@@ -0,0 +1,147 @@
+This means a vast quantity of programmers will be potentially exposed to concurrency issues.
+Bringing an ownership model to Python will help to make it easier to write concurrent programs, and to avoid the pitfalls of concurrency.
+
+This is a perfect opportunity to influence the future of Python, and to help make programming experience better for everyone.
+
+### Why not threads and locks?
+
+We are initially exploring using message passing between sub-interpreters.  However, we could also apply the region ideas to locks, and provide a lock that gives temporary access to a region of memory associated with that lock.
+
+We have sketched out a design for how to integrate with the full PEP 703 work, but have not attempted to implement that yet.
+
+### Why not simply use Rust's ownership model?
+
+Rust's ownership model is designed for a statically typed language, and restricts the type of object graphs that can be used in the language.
+There is so much existing code in Python that the restrictions of Rust's ownership model would be too limiting.
+We are designing a new ownership model based on regions that is designed to work with Python's dynamic typing and existing object graphs.
+
+Our approach draws heavily from the experience of languages with ownership models like Rust, Cyclone Encore, and Pony, but is based completely on dynamic checks.  This completely alters the kind of things that can be checked in comparison to a statically typed approach.
+
+### What has Project Verona learnt from Pyrona?
+
+Performing ownership research in a dynamically typed language has been a massive learning experience for the Project Verona team.
+It has challenged every assumption we have made about ownership and concurrency in a statically typed language.
+Moreover, it has shown us certain programming patterns that are really easy to check in a dynamic region system, but are really hard to check in a statically typed language.
+We are currently re-evaluating the balance between the static and dynamic checks in our ownership model, and how to best combine them.  Hopefully, we will be able to bring some of these ideas back into the Project Verona research language.
+
+### Where can I find out more?
+
+We have a detailed list of publications related to Project Verona on our [publications page](/publications.html).
+The most relevant paper to Pyrona is [Dynamic Region Ownership for Concurrency Safety](https://www.microsoft.com/en-us/research/publication/dynamic-region-ownership-for-concurrency-safety/).
+
+You can try our toy language, [FrankenScript](https://github.com/fxpl/frankenscript).
+
+We are also working on several forks of Python that implement our ideas, these are currently on [GitHub](https://github.com/mjp41/cpython).
+
+
+### How can I get involved?  Who can I talk to?
+
+We have listed a few places where you can find parts of the project that are available.
+If you want to find out more about those, raise issues on the appropriate GitHub repo. 
```

Comment:
Should we also throw in an email address in case they feel like an issue is "overkill" or "too public"?